tests/integration/cli/test_run.py

```diff
@@ -31,6 +31,17 @@ def test_run(ape_cli, runner, project):
             assert "Super secret script output" in result.output
 
 
+@skip_projects_except("script")
+def test_run_subdirectories(ape_cli, runner):
+    result = runner.invoke(ape_cli, ["run"])
+    assert result.exit_code == 0, result.output
+    # By default, no commands are run
+    assert "Super secret script output" not in result.output
+    result = runner.invoke(ape_cli, ["run", "subdirectory", "subdirectory_deploy"])
```

Comment:
After adding the other kinds, we will want to run them here as well.

Reply:
I added some more to the subdirectory folder and then am running them with the list comprehension. Does this meet that requirement or do I need to add more subdirectories as well?

Reply:
This suffices! But make sure to indent the assertions so that we properly test each script:
https://github.com/ApeWorX/ape/pull/1185#discussion_r1072837966
https://github.com/ApeWorX/ape/pull/1185#discussion_r1072838146